README.md

```diff
@@ -90,10 +90,13 @@ The [Helm chart](https://artifacthub.io/packages/helm/metrics-server/metrics-ser
 
 ## Security context
 
-Metrics Server requires the `CAP_NET_BIND_SERVICE` capability in order to bind to a privileged ports as non-root.
-If you are running Metrics Server in an environment that uses PSPs or other mechanisms to restrict pod capabilities, ensure that Metrics Server is allowed
-to use this capability.
-This applies even if you use the `--secure-port` flag to change the port that Metrics Server binds to to a non-privileged port.
+Metrics Server may require the `CAP_NET_BIND_SERVICE` capability, in order to bind to a privileged port as non-root.
```

Comment:
Word `may` is not correct, we know precisely which Metrics Server versions require it and which not.
Answer is that older Metrics Server versions like v0.4.0-v0.4.4 and v0.5.0-v0.5.1 require `CAP_NET_BIND_SERVICE`.

Reply:
> Word `may` is not correct, we know precisely which Metrics Server versions require it and which not. Answer is that older Metrics Server versions like v0.4.0-v0.4.4 and v0.5.0-v0.5.1 require `CAP_NET_BIND_SERVICE`.

It depends on the end user's environment and the way they configure Metrics Server, does it not? The "older versions" require `CAP_NET_BIND_SERVICE` _regardless_ of whether or not they bind to a privileged port. I'm not sure how relevant it is, but I wrote `may` with a newer version e.g. 0.5.2 being used in an older configuration in mind. Best practice would be to use a non-privileged port, but I was anticipating users may opt to continue to use the same port they've been using until now.

Reply:
If you'd like to suggest an alternative phrasing, I'll update the PR verbatim @serathius